lxd/storage/drivers/driver_zfs_volumes.go

```diff
@@ -31,6 +31,9 @@ import (
 	"github.com/lxc/lxd/shared/validate"
 )
 
+// zfsMaxVolBlocksize is a maximum value for volblocksize property.
+const zfsMaxVolBlocksize = 128 * 1024
```

Comment:
I'd keep that const with the others in driver_zfs.go

Reply:
Fixed